man/ostree-find-remotes.xml

```diff
@@ -0,0 +1,113 @@
+Boston, MA 02111-1307, USA.
+-->
+
+<refentry id="ostree">
+
+    <refentryinfo>
+        <title>ostree find-remotes</title>
+        <productname>OSTree</productname>
+
+        <authorgroup>
+            <author>
+                <contrib>Developer</contrib>
+                <firstname>Colin</firstname>
+                <surname>Walters</surname>
+                <email>walters@verbum.org</email>
+            </author>
+        </authorgroup>
+    </refentryinfo>
+
+    <refmeta>
+        <refentrytitle>ostree find-remotes</refentrytitle>
+        <manvolnum>1</manvolnum>
+    </refmeta>
+
+    <refnamediv>
+        <refname>ostree-find-remotes</refname>
+        <refpurpose>Find remotes to serve the given refs</refpurpose>
+    </refnamediv>
+
+    <refsynopsisdiv>
+            <cmdsynopsis>
+                <command>ostree find-remotes</command> <arg choice="opt" rep="repeat">OPTIONS</arg> <arg choice="req">COLLECTION-ID</arg> <arg choice="req">REF</arg> <arg choice="opt" rep="repeat">COLLECTION-ID REF</arg>
+            </cmdsynopsis>
+    </refsynopsisdiv>
+
+    <refsect1>
+        <title>Description</title>
+
+        <para>
+            OSTree has the ability do pulls not just from configured remote
+            servers but also from peer computers on the LAN and from mounted
+            filesystems such as USB drives. Currently this requires compiling
+            OSTree with experimental API enabled, and it requires the use of
+            Collection IDs and GPG verification.
+        </para>
+        <para>
+            The <command>find-remotes</command> command searches for remotes
+            which claim to provide one or more of the given COLLECTION-ID REF
+            pairs and prints information about them, with remotes sorted by
+            latency (Mounts > LAN > Internet). By default, OSTree searches for
+            remotes in configuration files, on mounted filesystems (in a
+            well-known location), and on the LAN using Avahi. Searching for LAN
+            remotes requires OSTree to have been compiled with Avahi support,
+            and it requires an Avahi daemon to be running. You can override the
+            default set of finders (sources for remotes) using the
+            <option>--finders</option> option documented below.
+        </para>
+    </refsect1>
+
+    <refsect1>
+        <title>Options</title>
+
+        <variablelist>
+            <varlistentry>
+                <term><option>--disable-fsync</option></term>
+
+                <listitem><para>
+                  Do not invoke fsync().
+                </para>
+                </listitem>
+            </varlistentry>
+
+            <varlistentry>
+                <term><option>--finders</option>=FINDERS</term>
+
+                <listitem><para>
+                  Use the specified comma separated list of finders rather than
+                  the default set. Possible values: config, lan, mount (or any
```

Comment:
`config`, `lan` and `mount` should probably be [`<literal>`](http://tdg.docbook.org/tdg/4.5/literal.html) here.